pyproject.toml

```diff
@@ -26,7 +26,10 @@ nbformat = "^5.1.3"
 nbconvert = "^6.3.0"
 requests = "^2.26.0"
 jsonschema = "^3.0.1"
-numpy = "^1.21.4"
+numpy = [
+    {version = "^1.21.4,<1.22", python = "<3.8"},
+    {version = "^1.22", python = ">=3.8"}
+]
```

Comment:
Had to do this because poetry is not smart enough to lock different numpy versions on different python versions by itself.